rpcs3/Emu/Cell/lv2/sys_net/lv2_socket_native.cpp

```diff
@@ -106,6 +118,8 @@ std::tuple<bool, s32, std::shared_ptr<lv2_socket>, sys_net_sockaddr> lv2_socket_
 	::sockaddr_storage native_addr;
 	::socklen_t native_addrlen = sizeof(native_addr);
 
+	feign_tcp_conn_failure = false; // TODO
```

Comment:
Not needed, disconnecting happens before the socket is destroyed no matter what(it happens in sys_net_bnet_close).